build.sbt

```diff
@@ -107,10 +97,10 @@ val MUnitFlakyOK   = sys.env.get("MUNIT_FLAKY_OK") match {
 lazy val tests = crossProject(JVMPlatform, JSPlatform)
   .crossType(CrossType.Full)
   .in(file("modules/tests"))
+  .enablePlugins(NoPublishPlugin)
```

Comment:
Is this typelevel?

Reply:
Yes.

Reply:
Yes.
https://github.com/typelevel/sbt-typelevel/blob/main/no-publish/src/main/scala/org/typelevel/sbt/NoPublishPlugin.scala